gemfiles/kubeclient_4.gemfile

```diff
@@ -2,6 +2,6 @@
 
 source "https://rubygems.org"
 
-gem "kubeclient", "4.0.0"
+gem "kubeclient", "4.12.0"
```

Comment:
Bumped to latest due to vulnerability note: 
https://github.com/ManageIQ/kubeclient?tab=readme-ov-file#vulnerability-in--v492